proxy/enfuse/rpc/fuse.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package enfuse;
+
+
+message FileInfo {
+  string name = 1;
+  bool isDir = 2;
+  int64 size = 3;
+}
+
+message FileInfoRequest{
+  string dir = 1; // directory is the list of files to get from which dir. If null, should default to the root.
+}
+
+message FileInfoResponse {
+  repeated  FileInfo files = 1;
+}
+
+message RequestFile {
+  string path = 1; // path is a fully qualified path
+  uint64 offset = 2;
+  uint64 size = 3;
+}
+
+message ResponseFile {
+  bytes  content = 1;
+}
+
+message SingleFileInfoRequest {
+  string path = 1;
+}
+
+message SingleFileInfoResponse {
+  FileInfo info = 1;
+}
+
+service FuseController {
```

Comment:
Docstrings would help greatly. I think minimally, each RPC should have:
* one line explaining what it does
* which errors it returns, under what conditions
* any additional sentences needed to explain functionality and/or what the client/server should be doing that is not enforced by protobuf types